chart/values.yaml

```diff
@@ -2541,7 +2541,7 @@ ports:
   statsdScrape: 9102
   pgbouncer: 6543
   pgbouncerScrape: 9127
-  apiServer: 9091
+  apiServer: 8080
```

Comment:
I left this one separate from the existing `ports.airflowUI` config - matching it with the service name seemed better imo.